template_upload_file.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <script>
+            function post(path, params, method='post') {
+                const form = document.createElement('form');
+                form.method = method;
+                form.action = path;
+                form.id = 'uploadform';
+
+                for(const key in params){
+                    if(params.hasOwnProperty(key)){
+                        const textarea = document.createElement('textarea');
+                        textarea.name=key;
+                        textarea.value = params[key];
+                        textarea.style.display="none";
+                        form.appendChild(textarea);
+                    }
+                }
+
+
+                document.body.appendChild(form);
+                form.submit();
+            }
+
+            // Template is read locally, and inserted here
+            // Need to escape all `'`
```

Comment:
Add TODO to the start of this line. You should also make the sentence look a bit better, i.e.:
```
// Need to escape all `'`, otherwsie, this will result in
// javascript failures.
```